rpm/harbour-callrecorder.spec

```diff
@@ -79,7 +79,6 @@ systemctl-user daemon-reload
 # install
 if [ $1 = 1 ]; then
     echo "Enabling service..."
-    mkdir -p /home/nemo/.config/systemd/user/user-session.target.wants
```

Comment:
I'd still leave the mkdir but change it to post-user-session for safety. As it was mentioned before, this line was needed for fresh installations on previous versions of SailfishOS. The directory was not created by systemd if it hadn't existed, so the service could not start without any error messages. I don't know the current situation, so better leave it there